db/migrate/20160204080445_add_posts_original_updated_at.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostsOriginalUpdatedAt < ActiveRecord::Migration
+  def change
+    add_column :posts, :original_updated_at, :datetime
```

Comment:
We should copy updated_at to original_updated_at:

`execute "UPDATE posts SET original_updated_at = updated_at"`